colors/seoul256.vim

```diff
@@ -422,6 +422,7 @@ call s:hi('rubyCurlyBlockDelimiter', [144, 101], ['', ''])
 call s:hi('rubyPredefinedIdentifier', [230, 52], ['', ''])
 " hi rubyRegexpSpecial
 
+hi Cursor ctermfg=NONE ctermbg=NONE cterm=inverse guifg=NONE guibg=NONE gui=inverse
```

Comment:
Thanks, can you check if we can simplify the line to `hi Cursor cterm=inverse gui=inverse`?